### ISSUE TITLE
Crash on any ragdoll

### ISSUE THREAD

**Issue by Midlex** [Sixze/ALS-Refactored#413]
As soon as I press X to ragdoll or fall from a high place unreal crashes, I have an empty project not sure what's wrong

I'm on ue 5.3
and the plugin version is 4.13

this is the crash report
```
Unhandled Exception: EXCEPTION_ACCESS_VIOLATION reading address 0x0000000000000178

UnrealEditor_Engine
UnrealEditor_ALS!AAlsCharacter::StartRagdollingImplementation() [...\Plugins\ALS-Refactored-4.13\Source\ALS\Private\AlsCharacter_Actions.cpp:760]
UnrealEditor_CoreUObject
UnrealEditor_CoreUObject
UnrealEditor_Engine
UnrealEditor_ALSExtras!TBaseUObjectMethodDelegateInstance<0,AAlsCharacterExample,void __cdecl(void),FDefaultDelegateUserPolicy>::Execute() [...\Engine\Source\Runtime\Core\Public\Delegates\DelegateInstancesImpl.h:650]
UnrealEditor_ALSExtras!TEnhancedInputUnifiedDelegate<TDelegate<void __cdecl(void),FDefaultDelegateUserPolicy> >::Execute<>() [...\Engine\Plugins\EnhancedInput\Source\EnhancedInput\Public\EnhancedInputComponent.h:110]
UnrealEditor_EnhancedInput
```

**Comment by Midlex** [comment]
This only happens when creating a child of the B_Als_Character, using AlsCharacterExample works

**Comment by itsaust** [comment]
I am having the same issue :( any workarounds here?

**Comment by itsaust** [comment]
the parent of my character is AlsCharacterExample and im still crashing with the same error as soon as ragdoll triggers

**Comment by Midlex** [comment]
Yeah im now getting a crash using AlsCharacterExample too, not sure what changed
> the parent of my character is AlsCharacterExample and im still crashing with the same error as soon as ragdoll triggers. also making my character a child of B_Als_Character i also get the crash.

**Comment by Sixze** [comment]
I tried inheriting from both `AAlsCharacterExample` and `B_Als_Character` and did not encounter this issue.

Can you please download the "Editor symbols for debugging" in Epic Games Launcher and provide me with more detailed text from the crash reporter?

**Comment by Qianfulong** [comment]
I also had the crash issue on my custom character BP - which was simple copy of B_Als_Character - and in my case the reasopn for the crash was a missing physics asset. So make sure to have one in the mesh settings.
![PA](https://github.com/Sixze/ALS-Refactored/assets/6871003/b069cc78-f14e-4268-9a1d-e5608f63b9f0)

**Comment by Sixze** [comment]
Fixed in db65de5ebe10e2b91b4122154ef4f739a47ffb91.
